### ISSUE TITLE
Hotkey in HTML

### ISSUE THREAD

**Issue by CreatorEzio214** [G33kDude/Neutron.ahk#19]
Is there a way to have the Hotkey Object from AHK GUI to show on Neutrons UI?

Gui, Add, Hotkey, x10 y10 w100 <--- this in HTML

**Comment by mfaizsyahmi** [comment]
Nah. HTML only has what it has as per standard. You have to code it yourself, or find an external lib that adds something similar (and pray it still supports IE11).

One idea is to add an ahk hotkey gui element right over where you want it to be. You can use neutron's `hwnd` property which is also valid for ahk's gui commands:
```autohotkey
    Gui, % neutron.hWnd ":Add", Hotkey, x10 y10 w100
```

If your HTML has pages/tabs then it's up to you to figure out how to show\hide the hotkey control in concert with your html content, by adding event listeners that call the `ahk` object etc. You probably also have to prevent your window from resizing because you surely won't be able to reflow your ahk gui alongside the HTML as easily.
